src/Utils/JsLoader.php

```diff
@@ -13,32 +13,31 @@
  */
 class JsLoader
 {
-    /** @var string Javascript file location. */
-    public static $cdn = 'https://cdn.jsdelivr.net/gh/atk4/google-address';
+    private static bool $isLoaded = false;
 
-    /** @var string Javascript file version. */
-    public static $version = '2.2.2';
+    /** The Google api developer key. */
+    protected static string $apiKey = '';
 
-    /** @var bool */
-    private static $isLoaded = false;
+    /** Default location of js file, use this if $locationUrl is not specified in JsLoader::load($app, '/new/path/to/js-file.min.js'); */
+    protected static string $defaultLocationUrl = '/assets/atk-google-maps.min.js';
```

Comment:
should use https://github.com/atk4/ui/blob/8f6d4deaa8a489d0a8af3b23ac6fb881f4c34be5/src/App.php#L635